### PR TITLE
TV Linux app - Move audio output and media playback cluster to endpoint 1

### PR DESCRIPTION
**Pull request by lazarkov**
#### Problem
- Audio Output cluster is on endpoint 2, but it should be on endpont 1
- There is no Media Playback cluster on endpoint 1

#### Change overview
see above

#### Issue(s) fixed:
[Boris comment](https://github.com/project-chip/connectedhomeip/pull/18191)

#### Testing
Run `./scripts/run_in_build_env.sh \  
                  "./scripts/tests/run_test_suite.py \
                     --chip-tool ./out/debug/standalone/chip-tool \
                     run \
                     --iterations 1 \
                     --all-clusters-app ./out/debug/standalone/chip-all-clusters-app \
                     --tv-app ./out/debug/standalone/chip-tv-app \
                  "`

### CODE REVIEW

**Review by chrisdecenzo — APPROVED** [review]

**Review by bzbarsky-apple — APPROVED** [review]

**Comment by bzbarsky-apple** [comment]
@lazarkov Needs merge

File: examples/tv-app/tv-common/tv-app.matter [hunk 1]
```diff
@@ -2340,6 +2343,9 @@ endpoint 1 {
     ram      attribute clusterRevision default = 1;
   }
 
+  server cluster MediaPlayback {
```
Comment (andy31415) [inline examples/tv-app/tv-common/tv-app.matter:2346]: why does this empty cluster exist?

Comment (andy31415) [reply examples/tv-app/tv-common/tv-app.matter:2346]: In another place in this file we have:

```
server cluster MediaPlayback {
    ram      attribute currentState;
    ram      attribute startTime default = 0xFF;
    ram      attribute duration;
    callback attribute sampledPosition;
    ram      attribute playbackSpeed;
    ram      attribute seekRangeEnd;
    ram      attribute seekRangeStart;
    ram      attribute featureMap default = 0x0003;
    ram      attribute clusterRevision default = 1;
  }
  ```
  
  So wondering here why these are missing. At least featureMap and clusterRevision should probably exist.

Comment (lazarkov) [reply examples/tv-app/tv-common/tv-app.matter:2346]: Hey @andy31415 I've created a PR that fixes this. I've forgot to enable attributes & commands for the given endpoint

https://github.com/project-chip/connectedhomeip/pull/18762

**Pull request merged**